app/main.py

```diff
@@ -1,2 +1,49 @@
+INITIAL_CAPACITY = 8
+LOAD_FACTOR = 2 / 3
```

Comment:
The `LOAD_FACTOR` is defined as `2 / 3`, which is a float. Ensure that this is the intended behavior, as it might affect how the load factor is used in calculations.